process/rating/ratingsData.go

```diff
@@ -510,6 +515,10 @@ func getRatingStepsForEpoch(epoch uint32, ratingStepsPerEpoch []config.RatingSte
 }
 
 func getChainParamsForEpoch(epoch uint32, chainParamsList []config.ChainParametersByEpochConfig) (config.ChainParametersByEpochConfig, bool) {
+	slices.SortFunc(chainParamsList, func(a, b config.ChainParametersByEpochConfig) bool {
```

Comment:
Sorts inline (alters input array) I think, but should be fine (I assume no concurrency).